### PR TITLE
[pleasereview] introduce solium and solve all linter/compiler errors/warnings (#59)

### LINKED ISSUE #59: fix solidity compilation warnings

### PR DESCRIPTION
**Pull request by snd**
adds solium, fixes all solium errors, runs solium on travis, fixes all solc compilation warnings.

resolves #59 and #68 

should not change any functionality.

~~depends on `package.json` introduced in https://github.com/paritytech/parity-bridge/pull/70 to be merged~~

### CODE REVIEW

**Comment by snd** [comment]
done. can i get a review?

File: contracts/bridge.sol [hunk 1]
```diff
@@ -152,6 +159,7 @@ contract HomeBridge {
     }
 }
 
+
```
Comment (tomusdrw) [inline contracts/bridge.sol]: Seems redundant.

Comment (snd) [reply contracts/bridge.sol]: it's a default solium rule: http://solium.readthedocs.io/en/latest/user-guide.html#list-of-core-rules
search for rule `blank-lines`: `Ensure that there is exactly a 2-line gap between Contract and Function declarations`

i think it increases readability a bit

Comment (tomusdrw) [reply contracts/bridge.sol]: I see, fair enough.

**Pull request merged**